pydarkstar/auction/manager.py

```diff
@@ -85,6 +87,26 @@ def buy_items(self, itemdata):
                                     if row.price <= data.price12:
                                         date = timeutils.timestamp(datetime.datetime.now())
                                         self.buyer.buy_item(row, date, data.price12)
+                                        
+                                        dboxCount = session.query(DeliveryBox).filter(
+                                            DeliveryBox.charname == row.seller_name,
+                                        ).count()
+                                        
+                                        dboxSend = DeliveryBox(
+                                            charid      = row.seller,
+                                            charname    = row.seller_name,
+                                            box         = 1,
+                                            slot        = dboxCount,
+                                            itemid      = 65535,
+                                            itemsubid   = 0,
+                                            quantity    = row.sale,
+                                            senderid    = 0,
+                                            sender      = "AH-Jeuno",
+                                            received    = 0,
+                                            sent        = 0,
+                                                    )
+                                        session.add(dboxSend)
+                                                    
```

Comment:
Ok, guessing this change is from @Korrbit then (the part I am interested in).

I think that maybe the buyer stopped sending money automatically to the delivery box when updating the SQL in darkstar and that this change fixes that behavior?  Just a guess.

Reply:
maybe. I'm not running the part where the bot buys player items currently, so I haven't seen any problems. I do know it was working way back when I did and that darkstars end hasn't changed. Last time we had a problem at all was when the expiry happened and we solved that by putting listing dates into the future I think.

Could be something you refactored broke the gil delivery or could be this person misunderstood a problem he was having. I'll check darkstar's discord, we get frequent questions about pydarkstar..

Reply:
nothing but "how do I use anaconda" questions, ¯\\\_(ツ)\_/¯

But looks like Korrbit based his fork off my fork instead of the main repo. 
![image](https://user-images.githubusercontent.com/6871475/65088914-181fe080-d989-11e9-8126-58e1d205ff94.png)

Reply:
it just hit me that I _have_ seen this problem, but caused by a missing trigger in someones database and not pydarkstar.

Reply:
@korrbit can you verify all your DB [triggers ](https://github.com/DarkstarProject/darkstar/blob/master/sql/triggers.sql)are intact, this is def a problem in pydarkstar?